Scripting/UTM.sdef

```diff
@@ -534,6 +539,14 @@
               description="The port number to listen on when the interface is a TCP server."/>
         </record-type>
         
+        <record-type name="qemu argument" code="QeAr" description="QEMU argument configuration.">
+          <property name="argument string" code="ArSt" type="text"
+            description="The QEMU argument as a string."/>
+            
+          <property name="file urls" code="FlUr" type="file" list="yes"
```

Comment:
Is this actually useful? UTM makes no attempt to open up the sandbox for custom args which contains paths meaning that any path you pass in that's not an absolute path to something that it can already access will fail.

Reply:
Thanks for the explanation.
I kept it to accommodate all the fields of a Qemu Argument type variable. 

Since it is not used currently (permissions need to be taken care if we use it), I will remove it.

Reply:
Dropped support for file urls in qemu arguments in scripting interface.
( Side note: I initially kept it, with a view that we might  use it in future to programmatically configure directory sharing. Can I achieve that by simply passing directory path in args string or it has to be a URL..? Something to test.)

Thanks for reviewing.